Formula/m/meta-package-manager.rb

```diff
@@ -184,6 +184,9 @@ class MetaPackageManager < Formula
   end
 
   def install
+    inreplace "meta_package_manager/bar_plugin.py",
+              "#!/usr/bin/env python3",
+              "#!#{Formula["python@3.12"].bin}/python3"
```

Comment:
can you use `rewrite_shebang detected_python_shebang, xx`

Reply:
Oh!  I had no idea that existed!  Neat!